web/index.html

```diff
@@ -25,6 +25,10 @@
     <script>
       window.APP = {};
       window.APP.baseUrl = "{{.BaseUrl}}";
+
+      const browserPrefers = !(window.matchMedia !== undefined && window.matchMedia("(prefers-color-scheme: light)").matches);
+      const { darkTheme = browserPrefers } = JSON.parse(localStorage.getItem("settings")) || {};
```

Comment:
What happens on initial load with this if there's nothing in localStorage from before?